guide/asynchronous-invocation/src/stock_quote_data_backend/Module.md

```diff
@@ -0,0 +1,3 @@
+# Hello World
+
+Prints "hello world" to command line output
```

Comment:
```suggestion
Prints "hello world" as the output in the command line.
```

Reply:
@praneesha This is a generated code from `ballerina` command, shall we leave them as is?

Reply:
Will be fixed via https://github.com/ballerina-platform/ballerina-lang/pull/19273.